lib/config.js

```diff
@@ -1,14 +1,23 @@
-const nconf = require("nconf");
+const defaults = require("../config/default.json");
+
+// const nconf = require("nconf");
 
 // Load configuration
 // First look for overrides from the command line or environment variables
-nconf.argv().env();
+// nconf.argv().env();
```

Comment:
please remove commented code

Reply:
this is in progress work, unfortunately nconf was not working in the web, non-node context, I will try to get it working without nconf for now

Reply:
ok

Reply:
i'm removing it